tests/ha/control_pod/test_control_pod_restart.py

```diff
@@ -364,49 +364,55 @@ def test_failover_control_pod_kubectl(self):
                                                     log_prefix=self.test_prefix, skipcleanup=True)
         assert_utils.assert_true(resp[0], resp[1])
         LOGGER.info("Step 1: Performed WRITEs-READs-Verify with variable sizes objects.")
-
         LOGGER.info("Control pod %s is hosted on %s node", self.control_pod_name, self.control_node)
-
-        failover_node = self.system_random.choice([ele for ele in self.host_worker_list if ele !=
-                                                   self.control_node])
-        LOGGER.debug("Fail over node is: %s", failover_node)
-
-        LOGGER.info("Step 2: Failover control pod %s to node %s and check cluster status",
-                    self.control_pod_name, failover_node)
-        pod_yaml = {self.control_pod_name: self.modified_yaml}
-        resp = self.ha_obj.failover_pod(pod_obj=self.node_master_list[0], pod_yaml=pod_yaml,
-                                        failover_node=failover_node)
+        LOGGER.info("Step 2: Taint the control node %s and delete control pod %s",
+                    self.control_node, self.control_pod_name)
```

Comment:
Should we choose control pod randomly as we will have multiple control pods now.

Reply:
No for this test, I'm assuming default config where we have only one control pod running and it will failover to another once tainted and restarted